src/screens/Account.jsx

```diff
@@ -424,15 +425,15 @@ export class ConfirmDatabaseDeletionModal extends Component {
             <Form onSubmit={(e) => this.deleteDB(e, db, ip)}>
               {openApiServer && (
                 <Modal.Body>
-                  <p>Please, enter "confirm" to DELETE ledger.</p>
+                  <p>Please, type CONFIRM to delete this ledger.</p>
                   <FormGroup>
                     <FormControl
                       onMouseDown={(e) => {
                         e.stopPropagation();
                       }}
                       type="text"
-                      value={this.state.confirmDeletion} 
-                      placeholder="Enter 'confirm' to DELETE ledger"
+                      value={this.state.confirmDeletion}
+                      placeholder="Enter CONFIRM to delete ledger"
```

Comment:
Can you change this placeholder to "Type CONFIRM to delete the ledger"?

Reply:
Done